packages/ember-runtime/lib/mixins/sortable.js

```diff
@@ -21,6 +21,8 @@ import {
   observer
 } from "ember-metal/mixin"; //ES6TODO: should we access these directly from their package or from how their exposed in ember-metal?
 
+export var sortableMixinDeprecation = 'Ember.SortableMixin is deprecated. Please use Ember.computed.sort instead.';
```

Comment:
This might be a little tricky because `expectDeprecation` tries to match the exact deprecation thrown with string value.

Ultimately I followed the array controller deprecation pattern: https://github.com/emberjs/ember.js/blob/release-1-13/packages/ember-runtime/lib/controllers/array_controller.js#L20-L21
https://github.com/emberjs/ember.js/blob/release-1-13/packages/ember-runtime/tests/controllers/array_controller_test.js#L14

I'm happy to continue to pursue this. Do you know of any other deprecations off hand that include `this` so I can follow that pattern for testing?